docs/source/index.rst

```diff
@@ -11,6 +11,7 @@ PyKEEN
    tutorial/first_steps
    tutorial/models
    tutorial/representations
+   tutorial/interactions
```

Comment:
there's a subsection in the models tutorial on how to construct a generic ER model using your own interaction function. Maybe you either want to cross-link from that to the new tutorial, or move this over into the interactions tutorial